lib/keyResolver.js

```diff
@@ -1,11 +1,11 @@
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
-import didIo from 'did-io';
-import {driver} from 'did-method-key';
+import {didIo} from 'bedrock-did-io';
+import {driver} from '@digitalbazaar/did-method-key';
 
 // Config did-io to support did:key driver
-didIo.use('key', driver());
+didIo.use(driver());
```

Comment:
Since we're using `bedrock-did-io`, this can be removed as it is already being handled here https://github.com/digitalbazaar/bedrock-did-io/blob/main/lib/main.js#L19. We can remove '@digitalbazaar/did-method-key' from dependencies as well.

Reply:
Thanks, updated!